setup.py

```diff
@@ -17,7 +17,7 @@
 with open("CHANGELOG.md") as f:
     CHANGELOG = f.read()
 
-install_requires = ["anyconfig", "appdirs", "docopt", "pyyaml", "requests"]
+install_requires = ["anyconfig", "appdirs", "docopt", "pyyaml", "requests", "tabulate"]
```

Comment:
If we have a requirements file, then this should be added there as well.

Reply:
Currently does not have a `requirements.txt`, out of scope for now.